gallery/src/components/NxNumberInput/NxNumberInputDisabledExample.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { NxNumberInput } from '@sonatype/react-shared-components';
+
+export default function NxNumberInputDisabledExample() {
+  return (
+    <>
+      <div>
+        <NxNumberInput placeholder="" value="" isPristine={true} disabled/>
+      </div>
+      <div>
+        <NxNumberInput placeholder=""
+                       value="42"
+                       isPristine={false}
+                       validatable={true}
+                       validationErrors={null}
+                       disabled/>
```

Comment:
Is there value in having both of these?

Reply:
> Is there value in having both of these?

I had a reason but I can't figure out what it was so I have removed it: https://github.com/sonatype/sonatype-react-shared-components/pull/486/commits/13520d93d897f406c7b155cbb0c0278773034b05